src/main/java/teammates/common/util/EmailType.java

```diff
@@ -13,6 +13,7 @@
     FEEDBACK_CLOSED("TEAMMATES: Feedback session closed [Course: %s][Feedback Session: %s]"),
     FEEDBACK_PUBLISHED("TEAMMATES: Feedback session results published [Course: %s][Feedback Session: %s]"),
     FEEDBACK_UNPUBLISHED("TEAMMATES: Feedback session results unpublished [Course: %s][Feedback Session: %s]"),
+    STUDENT_EMAIL_CHANGED("TEAMMATES: Summary of all Feedback Sessions [Course: %s]"),
```

Comment:
`Summary of course [%s][Course ID: %s]` is more like it, because the course join link is included and it certainly does not fit into the "all feedback sessions" category.